selfdrive/controls/controlsd.py

```diff
@@ -378,7 +378,7 @@ def update_events(self, CS):
       self.events.add(EventName.vehicleModelInvalid)
     if not self.sm['lateralPlan'].mpcSolutionValid:
       self.events.add(EventName.plannerError)
-    if not (self.sm['liveParameters'].sensorValid or self.sm['liveLocationKalman'].sensorsOK) and not NOSENSOR:
+    if not (self.sm['liveParameters'].sensorValid or self.sm['liveLocationKalman'].sensorsOK or self.sm['liveLocationKalman'].inputsOK) and not NOSENSOR:
```

Comment:
`inputsOK` captures a lot more than sensors (e.g. `carState` going invalid). we need to be very careful about alert text; this seems like it deserves its own alert